src/scripts/main.js

```diff
@@ -1 +1,59 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve();
+  });
+
+  setTimeout(() => reject(new Error('First promise was rejected')), 3000);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', (e) => {
+    if (e.button === 0 || e.button === 2) {
+      resolve();
+    }
+  });
+});
+
+const promise3 = Promise.all([
+  new Promise(resolve => {
+    document.addEventListener('click', () => {
+      resolve();
+    });
+  }),
+
+  new Promise(resolve => {
+    document.addEventListener('mousedown', (e) => {
+      if (e.button === 2) {
+        resolve();
+      }
+    });
+  }),
+]);
+
+function createElement(className, message) {
+  body.insertAdjacentHTML('beforeend', `
+  <div data-qa="notification" class="${className}">${message}</div>
+`);
+}
+
+promise1
+  .then((result) => {
+    createElement('success', 'First promise was rejected');
+  })
```

Comment:
You do almost the same for success for all 3 promises. Create a method for it, maybe